modules/adagioBidAdapter.js

```diff
@@ -23,7 +23,7 @@ pV6EP3MTLosuUEpLaQIDAQAB
 export function getAdagioScript() {
   try {
     const w = utils.getWindowTop();
-    const ls = w.localStorage.getItem(ADAGIO_LOCALSTORAGE_KEY);
+    const ls = (utils.hasLocalStorage()) ? w.localStorage.getItem(ADAGIO_LOCALSTORAGE_KEY) : null;
```

Comment:
Hi, 

As `utils. getDataFromLocalStorage()` checks if localStorage is available and returns null if nothing is found, we can just use:

```
const ls = utils.getDataFromLocalStorage(ADAGIO_LOCALSTORAGE_KEY);
```

Also, we call localStorage twice later in our bidder : lines 37 and 51. For this lines, we should use:

```
utils.removeDataFromLocalStorage(ADAGIO_LOCALSTORAGE_KEY);
```

Thanks!

Reply:
@osazos - you can go ahead and make the change you've suggested.

Reply:
@bretg, I created #4978.
Honestly, I don't know if there is a better way to submit my change. Please fell free to tell me.

Reply:
@osazos - I merged your changes from #4978 -- will close that PR.